# Compiling Chromium 78 spams the console with unknown warnings.
# This can be stopped by adding "-Wno-unknown-warning-option" to the cflags.

--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -62,6 +62,7 @@ config("compiler") {
     "/bigobj",  # Some of our files are bigger than the regular limits.
     "/utf-8",  # Assume UTF-8 by default to avoid code page dependencies.
     "/Zc:twoPhase",
+    "-Wno-unknown-warning-option",
   ]
 
   # Force C/C++ mode for the given GN detected file type. This is necessary
